tests/input/gltf/test_gltf_vtk.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+import sys
+import typing
+
+from PIL import Image
+import pytest
+
+from preview_generator.preview.builder.cad__vtk import ImagePreviewBuilderVtk
+from preview_generator.utils import ImgDims
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+CACHE_DIR = "/tmp/preview-generator-tests/cache/"
```

Comment:
Shouldn't we use a temporary directory (using mktemp)?

Reply:
probably. We need to refactor tests code.

Reply:
https://github.com/algoo/preview-generator/issues/243